### PR TITLE
Fixes #11844: Port some more techniques to multi-versionned format

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/11844

### CODE REVIEW

**Comment by ncharles** [comment]
Commit modified

**Comment by ncharles** [comment]
Commit modified

**Comment by ncharles** [comment]
Don't merge for now

**Comment by ncharles** [comment]
Commit modified

File: techniques/fileDistribution/copyGitFile/2.3/metadata.xml [hunk 1]
```diff
@@ -18,16 +18,34 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
     <OS version=">= 4 (Etch)">Debian</OS>
     <OS version=">= 4 (Nahant)">RHEL / CentOS</OS>
     <OS version=">= 10 SP1 (Agama Lizard)">SuSE LES / DES / OpenSuSE</OS>
     <AGENT version=">= 3.2.0">cfengine-community</AGENT>
   </COMPATIBLE>
 
-  <BUNDLES>
-    <NAME>download_from_shared_folder</NAME>
-  </BUNDLES>
+  <AGENT type="dsc">
```
Comment (amousset) [inline techniques/fileDistribution/copyGitFile/2.3/metadata.xml]: Isn't this a bug that should be fixed in previous version of the technique?

Comment (ncharles) [reply techniques/fileDistribution/copyGitFile/2.3/metadata.xml]: yes, see https://www.rudder-project.org/redmine/issues/11881

Comment (ncharles) [reply techniques/fileDistribution/copyGitFile/2.3/metadata.xml]: i'll do it in a separe issue, if it's ok with you

Comment (amousset) [reply techniques/fileDistribution/copyGitFile/2.3/metadata.xml]: :+1:

File: techniques/fileDistribution/copyGitFile/2.3/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+  <AGENT type="cfengine-community">
+    <BUNDLES>
+      <NAME>download_from_shared_folder_RudderUniqueID</NAME>
+    </BUNDLES>
+    <TMLS>
+      <TML name="copyFileFromSharedFolder"/>
+    </TMLS>
+  </AGENT>
+
+  <TMLS>
```
Comment (ncharles) [inline techniques/fileDistribution/copyGitFile/2.3/metadata.xml]: damn, there shouldn't be this TMLs

**Pull request merged**